<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialTracker</class>
 <widget class="QWidget" name="MaterialTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="LeftFrameStyleSheet">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background:rgba(64,64,64,60%);
    border: 1px solid black;
}

QPushButton
{
	background-color: rgba(136, 136, 136, 30%);
	color: white;
	min-width: 80px;
	border-style: solid;
	border-width: 1px;
	border-radius: 2px;
	border-color: black;
	padding: 1px;
}

QPushButton::flat
{
	background-color: transparent;
	border: none;
	color: #fff;
}

QPushButton::hover
{
	background-color: rgba(0, 39, 255, 20%);
	border: 1px solid black;

}

QPushButton:pressed 
{
    background-color: rgb(64, 64, 64);
}

QLabel
{
	color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="buttonVertLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="toolNameLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Material Tracker Tool</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addTabPushButton">
          <property name="text">
           <string>Add Tab</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updatePushButton">
          <property name="text">
           <string>Update Materials</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="sortCategoryPushButton">
          <property name="text">
           <string>Sort Category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sortNamePushButton">
          <property name="text">
           <string>Sort Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sortCurrentPushButton">
          <property name="text">
           <string>Sort Current</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sortGoalPushButton">
          <property name="text">
           <string>Sort Goal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sortPercentPushButton">
          <property name="text">
           <string>Sort Percent</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="savePushButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadPushButton">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileStatusLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
           </font>
          </property>
          <property name="text">
           <string>No File Loaded</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="tabsWidgetFrame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background:rgba(136,136,136,60%);
    border: 1px solid black;
}

QTabBar::tab 
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1,   			stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	border-style: solid;
	border-radius: 2px;
	border-width: 1px;
	border-color: black;
	border-bottom: none;
	padding: 5px;
	padding-left: 15px;
	padding-right: 15px;
}

QTabWidget::pane 
{
	background-color: rgba(0,0,0, 20%);
	border: 1px solid #666;
	top: 1px;
}

QTabBar::tab:last
{
	margin-right: 0; 
}

QTabBar::tab:!selected
{
	color: #b1b1b1;
	border-bottom-style: solid;
	background-color: rgb(32, 32, 32);
}

QTabBar::tab:selected
{
	margin-bottom: 0px;
}

QTabBar::tab:!selected:hover
{
	border-top-color: rgba(0, 39, 255, 20%);
}


 </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="materialsTabWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
